lib/bots/hackling.rb

```diff
@@ -0,0 +1,21 @@
+require "#{
```

Comment:
out of curiosity, is there a reason you need to put this code in a require statement?

i *could* download it and test and figure it out myself but... yeah.